CGPs/cgp-0066/mainnet.json

```diff
@@ -1,10 +1,23 @@
 [
   {
-    "contract": "GasPriceMinimum",
```

Comment:
There was an existing 66 mainnet.json but I'm not seeing any evidence that it was ever executed or associated with a CGP so 🤷‍♂️